content/shmem_and.tex

```diff
@@ -0,0 +1,44 @@
+\apisummary{
+  Atomically perform a non-fetching bitwise and operation on a
+  remote data object.
+}
+
+\begin{apidefinition}
+
+\begin{C11synopsis}
+void shmem_and(TYPE *dest, TYPE value, int pe);
+\end{C11synopsis}
+where \TYPE{} is one of the standard \ac{AMO} types specified by
+Table~\ref{stdamotypes}.
```

Comment:
My concern is that we are adding functions (e.g. shmem_long_and) that have implementation-defined behavior.  Portable applications should never use use these functions -- so why add them in the first place?